config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  resources :books, only: [:show]
+  resources :users, shallow: true do
+    resources :books, only: [:show]
+  end
```

Comment:
You don't need any routes except the one we gave you. All the user stories start with `When I visit the book show page`